framework/tests/purity-suggestions/Test.java

```diff
@@ -7,7 +7,6 @@
 // copied from Purity.java)
 
 // This warning is for the implicit constructor of class Test
```

Comment:
This should be deleted.

Reply:
I found the comment helpful because the warning is about implicit code.  If you think it's obvious, we could delete the comment.

Reply:
The warning doesn't occur anymore.